dockerfiles/theia/src/extensions.json

```diff
@@ -11,7 +11,7 @@
             "name": "che-theia-ssh-extension",
             "source": "https://github.com/eclipse/che-theia-ssh-plugin",
             "folder": "ssh-extension",
-            "checkoutTo": "master",
+            "checkoutTo": "0.0.0",
```

Comment:
could we have as tag like 0.0.1 ?

Reply:
Sure but let's set the new version in the extension: https://github.com/eclipse/che-theia-ssh-plugin/pull/10

Reply:
done